lib/statsd/instrument.rb

```diff
@@ -280,9 +284,13 @@ def method_visibility(method)
     end
   end
 
-  attr_accessor :logger, :default_sample_rate, :prefix
+  attr_accessor :logger, :default_sample_rate, :prefix, :default_tags
   attr_writer :backend
 
+  def default_tags=(tags)
+    @default_tags = tags ? StatsD::Instrument::Metric.normalize_tags(tags) : nil
```

Comment:
No need for this ternary operator, right? `normalize_tags` knows how to deal with `nil` already?